backend/src/test/kotlin/be/osoc/team1/backend/unittests/StudentControllerTests.kt

```diff
@@ -111,6 +112,30 @@ class StudentControllerTests(@Autowired private val mockMvc: MockMvc) {
         mockMvc.perform(get(editionUrl).principal(defaultPrincipal)).andExpect(status().isOk)
     }
 
+    @Test
+    fun `getAllStudents Basic view returns a basic form of students`() {
+        val testStudent1 = Student("L1", "VC", testEdition)
+        val testStudent2 = Student("L2", "VC", testEdition)
+        val testStudent3 = Student("L3", "VC", testEdition)
+        val testStudent4 = Student("L4", "VC", testEdition)
+        val allStudents = listOf(testStudent1, testStudent2, testStudent3, testStudent4)
+        every { studentService.getAllStudents(defaultSort, testEdition) } returns allStudents
+        mockMvc.perform(get("$editionUrl?view=Basic").principal(defaultPrincipal)).andExpect(status().isOk)
+            .andExpect(content().json(objectMapper.writerWithView(StudentView.Basic::class.java).writeValueAsString(PagedCollection(allStudents, 4))))
+    }
```

Comment:
Might be good to also check that only the expected values are returned, since you're only testing if the return value is the same as manually calling the view.